sc/social/like/tests/test_plugin_facebook.py

```diff
@@ -132,130 +131,27 @@ def test_privacy_plugin_view_html(self):
         view = portal.restrictedTraverse(plugin_view)
 
-        metadata = view.metadata()
-        self.assertIn('og:site_name', metadata)
-
-        # At root, use site logo
-        image_url = view.image_url()
-        self.assertIn('logo.png', image_url)
-
-        # At root, use website type
-        og_type = view.type()
-        self.assertIn('website', og_type)
-
-    def test_plugin_view_document(self):
-        plugin = self.plugin
-        document = self.document
-        portal = self.portal
-
-        plugin_view = plugin.view()
-        view = document.restrictedTraverse(plugin_view)
-
-        # At document, use site logo
-        image_url = view.image_url()
-        self.assertIn('logo.png', image_url)
-
-        # At document, use article type
-        og_type = view.type()
-        self.assertIn('article', og_type)
-
-        # At document, default page of portal, use website type
-        portal.setDefaultPage(document.id)
-        og_type = view.type()
-        self.assertIn('website', og_type)
-
-    def test_plugin_view_image(self):
-        plugin = self.plugin
-        image = self.image
-
-        plugin_view = plugin.view()
-        view = image.restrictedTraverse(plugin_view)
-
-        # At image, use local image
-        image_url = view.image_url()
-        self.assertNotIn('logo.png', image_url)
-        self.assertEqual(view.image_width(), 1024)
-        self.assertEqual(view.image_height(), 768)
-        self.assertEqual(view.image_type(), 'image/png')
-
-        # XXX: avoid failures because of unchanged modification date
-        #      this happens only on Dexterity-based content types
-        from time import sleep
-        sleep(1)
-
-        # Set a larger image
-        set_image_field(image, load_image(1920, 1080), 'image/png')
-
-        plugin_view = plugin.view()
-        view = image.restrictedTraverse(plugin_view)
-        self.assertEqual(view.image_width(), 1200)
-        self.assertEqual(view.image_height(), 675)
-
-    def test_plugin_view_image_large(self):
-        plugin = self.plugin
-        image = self.image
-        set_image_field(image, load_image(1920, 1080), 'image/png')
-
-        plugin_view = plugin.view()
-        view = image.restrictedTraverse(plugin_view)
-
-        # At newsitem, use image
-        image_url = view.image_url()
-        self.assertNotIn('logo.png', image_url)
-
-        self.assertEqual(view.image_width(), 1200)
-        self.assertEqual(view.image_height(), 675)
-
-    def test_plugin_view_newsitem(self):
-        plugin = self.plugin
-        newsitem = self.newsitem
-
-        plugin_view = plugin.view()
-        view = newsitem.restrictedTraverse(plugin_view)
-
-        # At newsitem, use image
-        image_url = view.image_url()
-        self.assertNotIn('logo.png', image_url)
-        self.assertEqual(view.image_width(), 1024)
-        self.assertEqual(view.image_height(), 768)
-
-    def test_plugin_view_newsitem_large(self):
```

Comment:
These tests are not needed anymore since we are using **scale large** by default for all plugins.